cdk/infrastructure/constructs/existing/igvf_dev.py

```diff
@@ -0,0 +1,36 @@
+from constructs import Construct
+
+from shared_infrastructure.igvf_dev.connection import CodeStarConnection
+from shared_infrastructure.igvf_dev.environment import US_WEST_2 as US_WEST_2
```

Comment:
Added for `mypy`. I think when run in `strict` mode it enables this `--no-implicit-rexport` flag: https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-implicit-reexport

From their example:
```
# This will re-export it as bar and allow other modules to import it
from foo import bar as bar
```
The other option is to set `--implicit-reexport` manually.